devtools/create-gossipstore.h

```diff
@@ -0,0 +1,11 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <common/amount.h>
+
+struct scidsat {
+	char scid[17];
+        struct amount_sat sat;
+} scidsat;
+
+struct scidsat * load_scid_file(FILE * scidfd);
```

Comment:
We don't do a source check on header files in the dev directory, which is a mistake.  Generally, they should
be protected against multiple inclusions with `#ifndef` guards.  But importantly, I don't think this needs to be in a header at all does it?